src/ontology/uberon-edit.obo

```diff
@@ -224207,6 +224207,43 @@ relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
 relationship: located_in UBERON:0002371 ! bone marrow
 property_value: dcterms-date "2023-06-26T13:48:30Z" xsd:dateTime
 
+[Term]
+id: UBERON:8480071
+name: skin of lobe of tail
+def: "A zone of skin that is part of the lobe tail." [https://orcid.org/0000-0002-0819-0473]
+synonym: "fluke skin" NARROW [PMID:21959845]
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0013131 ! lobe of tail
+property_value: dcterms-date "2023-08-29T09:37:23Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480072
+name: skin of shin
+def: "A zone of skin on the front hindlimb zeugopod." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0003823 ! hindlimb zeugopod
+property_value: dcterms-date "2023-08-29T09:47:32Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480073
+name: skin of scapula region
+def: "A zone of skin that is above the scapula in the upper back region." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
+relationship: dc-contributor https://orcid.org/0000-0002-0819-0473
+relationship: part_of UBERON:0008231 ! dorsal thoracic segment of trunk
+property_value: dcterms-date "2023-08-29T09:53:33Z" xsd:dateTime
+
+[Term]
+id: UBERON:8480074
+name: skin of gonad
+def: "A zone of skin part of some gonad." [https://orcid.org/0000-0002-0819-0473]
+is_a: UBERON:0000014 ! zone of skin
```

Comment:
Consider adding logical definition 'zone of skin' and ('part of' some gonad)

Reply:
@aleixpuigb OK. We need this term to describe the cell line GM01722 (CVCL_JC93) originated from a patient whose sex is ambiguous. "She is female but "X,Y".